stories/LineChart.stories.js

```diff
@@ -875,4 +902,23 @@ storiesOf('LineChart', module)
       }
     }
     return <LimitXSubDomain />;
-  });
+  })
+  .add('onMouseOut', () => (
+    <DataProvider
+      defaultLoader={staticLoader}
+      series={[{ id: 1, color: 'steelblue' }, { id: 2, color: 'maroon' }]}
+      xDomain={staticXDomain}
+      xSubDomain={[
+        Date.now() - 1000 * 60 * 60 * 24 * 30,
+        Date.now() - 1000 * 60 * 60 * 24 * 10,
+      ]}
+    >
+      <LineChart
+        height={CHART_HEIGHT}
+        onMouseOut={e => {
```

Comment:
nit: you can just do `onMouseOut={action('onMouseOut')}` since `action` returns a function